salt/metalk8s/orchestrate/downgrade/init.sls

```diff
@@ -1,4 +1,4 @@
-{%- set dest_version = pillar.orchestrate.dest_version %}
+{%- set dest_version = salt.metalk8s_kubernetes.namespace_annotations('kube-system', kubeconfig="/etc/kubernetes/admin.conf")['metalk8s.scality.com/cluster-version'] %}
```

Comment:
Move this line and use the variable define just bellow please

Reply:
And no btw you should retrieve it from the ext_pillar I think in this one `salt/_pillar/metalk8s_nodes.py` retrieve it like for others version and store I would say in `metalk8s.cluster_version`